src/email-sender.js

```diff
@@ -0,0 +1,39 @@
+const nodemailer = require('nodemailer');
+
+/*
+*                             HOW TO USE                                 *
+* Import this file - const sendEmail = require('./email-sender);         *
+* send email - sendEmail.sendMessage("Put your error message in here");  *
+*/
+
+exports.sendMessage = async function (errorMessage) {
+  // Recipients list of users to send the email too
+  // receipiants is commas separated (emailName@hotmail.com,emailName2@hotmail.com)
+  const receipiants = '';
+
+  // Credientials to send an email from
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: '', // Email Name (.env variables must be used refer to issue#60)
+      pass: '', // Email Pass (.env variables must be used refer to issue#60)
+    },
+  });
+
+  // Email Content
+  const mailOptions = {
+    from: '', // Email Name
+    to: receipiants,
+    subject: 'Seneca Telescope had an error', // Subject Line
+    html: `<p>An Error has occurred <br/>Error Message: ${errorMessage} </p>`, // Message Body
+  };
+
+  // Send the email with the email content
+  transporter.sendMail(mailOptions, (err, info) => {
+    if (err) {
```

Comment:
Hey @RyanWils ! Maybe we could return and err if it transporter.sendMail() doesn't work and a true or something else if it does. This would make it easier to test for future development. What do you think?

Reply:
Ya as you highlighted in your suggestion, the transporter sends the email and will print an error if it unsuccessful/print the information if it does not have an error. From my knowledge this is not reliable because just like the other frameworks to send emails, Nodemailer just send a message to the outlook server and if its received successfully by the outlook server it returns true. Even if the email does not send to the user, as long as it is received by the outlook server it will return true. This is where you tests come in, maybe we can find something that returns false if it cannot find an email.